website/docs/components/table/partials/code/component-api.md

```diff
@@ -1,38 +1,50 @@
 ## Component API
 
-The Table component itself is where most of the options will be applied. However, the child components can also be used if a custom implementation is needed.
+The Table component itself is where most of the options will be applied. However, the API for the child components are also documented here, in case a custom implementation is desired.
```

Comment:
Change to either:
"API for the child components are" => "API for the child components is"
-OR-
"API for the child components are" => "APIs for the child components are"